ci-operator/step-registry/openshift/e2e/test/qe/automated-release-disruptive/openshift-e2e-test-qe-automated-release-disruptive-chain.metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+	"path": "openshift/e2e/test/qe/automated-release-disruptive/openshift-e2e-test-qe-automated-release-disruptive-chain.yaml",
```

Comment:
Perhaps use `openshift/e2e/test/qe/automated-release/disruptive/xxxx`

Reply:
@liangxia make sense, I moved the chain to sub dir, can you review again, thx